src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml

```diff
@@ -125,6 +125,10 @@
 	<bean id="VendorCreditMemo-wireTransferValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.CreditMemoWireTransferValidation" abstract="true" />
 	<bean id="VendorCreditMemo-foreignDraftValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.CreditMemoForeignDraftValidation" abstract="true" />
 
+    <bean id="PaymentRequest-accountingLineAccessibleValidation"
+          class="org.kuali.kfs.module.purap.document.validation.impl.PaymentRequestAccountingLineAccessibleValidation"
+          abstract="true" p:dataDictionaryService-ref="dataDictionaryService" p:documentDictionaryService-ref="documentDictionaryService"/>
```

Comment:
THe kualico version of this overlay doesn't include documentDictionaryService-ref, can you confirm we want that

Reply:
ah, never mind, this isn't an extension, I shall merge

Reply:
yes, that is the missing part that I have added